api_app/connectors_manager/serializers.py

```diff
@@ -0,0 +1,60 @@
+from rest_framework import serializers
+
+from intel_owl import secrets as secrets_store
+
+
+DATA_TYPE_MAPPING = {
+    "number": (
+        int,
+        float,
+    ),
+    "string": str,
+    "bool": bool,
+}
+
+
+class ConnectorConfigSerializer(serializers.Serializer):
+    """
+    serializer for connectors from connector_config.json.
+    """
+
+    disabled = serializers.BooleanField()
+    description = serializers.CharField(max_length=512)
+    python_module = serializers.CharField(max_length=128)
+    config = serializers.JSONField()
+    secrets = serializers.JSONField()
+    verification = serializers.SerializerMethodField()
```

Comment:
What is this field for? I didn't really understand `SerializerMethodFields` and what they do...

Reply:
https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield

Reply:
Basically `verification` will be populated by calling the `get_verification(self, obj)` method. So it's a  dynamic field that gets it value from a function call.